### DISCUSSION TITLE
my profile is not showing up when searched by anyone

### DISCUSSION THREAD

**Discussion by verhaxity** [community/community#65925] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

my github readme repo isnt showing up on my profile
![Screenshot (50)](https://github.com/community/community/assets/66476388/5b4d10b1-171c-4066-abd0-fe7110368283)
![Screenshot (51)](https://github.com/community/community/assets/66476388/08dd8eda-a4b7-41e9-b524-cf18f599c199)

+ when i search my username from another account it shows no user found
![Screenshot (52)](https://github.com/community/community/assets/66476388/dd655f6b-ba49-408c-848c-d01cf7f8dbb7)

everything is public
profile readme file and its public

private profile is not enabled
![Screenshot (53)](https://github.com/community/community/assets/66476388/2cb0de9e-534e-4c18-b011-988d1b85acc8)

**Comment by cehfisher** [comment]
Hi @verhaxity - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by davevad93** [comment]
Hi @verhaxity ,

That's odd because I found you by searching for your name and I can see your profile's README too:

![1](https://github.com/community/community/assets/94308234/6b6e7fdf-e3cd-4ae9-8fb7-d4ae83450e28)

![2](https://github.com/community/community/assets/94308234/f627e2fa-4fbd-4658-8f30-6e3ff22ca595)

**Reply by verhaxity** [reply, marked as answer]
the issue is resolved. this ticket was submitted previous year

**Reply by KHab1bull0** [reply]
how did you solve the problem

**Reply by A-Abduvohid** [reply]
After changing my github username,
my profile and repositories are not visible to others.
I do not know. why?
how did you solve the problem

**Reply by saulsantana0412** [reply]
> the issue is resolved. this ticket was submitted previous year

I have the same issue. Other people can't see my profile; even the public README doesn't show. What should I do?

**Reply by nichorvath** [reply]
I have the same issue. I cannot even clone my public repository from another profile.

**Comment by barismutluu** [comment]
Creating Your First Repository
After creating your account, you can upload your first project or files by creating a new repository:

1.Go to GitHub Homepage: On the GitHub homepage, click on the "+" icon in the upper right corner and select "New repository."
2.Fill in Repository Details: Enter the repository username and description.
3.Initialize with a README: Check the "Initialize this repository with a README" option to create a README file when the repository is created.
4.Create Repository: Click the "Create repository" button.

**There is a good explanation and video in the link below that will help you.**
https://github.com/kautukkundan/Awesome-Profile-README-templates?tab=readme-ov-file

**Comment by SidhantSaxena** [comment]
I am facing the same issue. Other accounts are not able to view my username when trying to give repo access.
what should i do?

**Reply by saulsantana0412** [reply]
Were you able to solve it? I have the same problem

**Comment by PaOneAdk** [comment]
same here

**Comment by Arhaan-Siddiquee** [comment]
@verhaxity How much time it took the Github Support to resolve this issue?
